15-may-17/solutions/simon/test/look_and_say_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule LookAndSayTest do
+  use ExUnit.Case
+  doctest LookAndSay
+
+  test "generate_look_and_say_sequence/1 with '1'" do
+    sequence = "1" |> LookAndSay.generate_look_and_say_sequence
```

Comment:
It's generally considered better style to avoid using the pipe (`|>`) operator where not necessary.

See: https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines